### PR TITLE
Scepter Blue: Various Fixes

### PR DESCRIPTION
**Pull request by m-laniakea**
Everything should be fine with the Blue line.

Corrected quest issues.
Modified Number Two's immunities.
Added Narain's Letter.

Awaiting testing for Maws.

### CODE REVIEW

File: src/scripts/world/boss_anachronos.cpp [hunk 1]
```diff
@@ -95,6 +99,28 @@ struct boss_anachronosAI : public ScriptedAI
...
+    if (quest->GetQuestId() != QUEST_WRATH_OF_NEPTULON)
+        return false;
+
+    if (npc->GetEntry() != NPC_ANACHRONOS)
+        return false;
+
+    if (!player)
+        return false;
+
+    std::string letterText = player->GetSession()->GetMangosString(LANG_NARAIN_LETTER);
```
Comment (Cgbd42) [inline src/scripts/world/boss_anachronos.cpp]: I think you should use script_text here

Comment (m-laniakea) [inline src/scripts/world/boss_anachronos.cpp]: How come? You could be right, but I'd like to know why.

**Comment by gheyhardy** [comment]
lmao hurry up unpaid dimwits they have a deadline to meet

File: src/scripts/world/boss_anachronos.cpp (hunk 1 above)
Comment (Cgbd42) [inline src/scripts/world/boss_anachronos.cpp]: Mangos string is more for core utils, like innkeeper gossips, bg's strings etc..
Here you are on script side (src/scripts), so imo script_text is more appropriated.

**Pull request merged**